### PR TITLE
i3s-debug: debug panel refactoring

### PR DESCRIPTION
**Pull request by belom88**
- refactoring for DebugPanel class;
- reset debug options on tileset change - it was request from @Tamrat-B

### CODE REVIEW

**Review by dryabinin-actionengine — APPROVED** [review]

File: examples/experimental/i3s-17-and-debug/app-debug.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ import {
   CONTRAST_MAP_STYLES,
   INITIAL_TILE_COLOR_MODE,
   INITIAL_BOUNDING_VOLUME_COLOR_MODE,
-  BOUNDING_VOLUME_TYPE
+  BOUNDING_SPHERE
```
Comment (ibgreen) [inline examples/experimental/i3s-17-and-debug/app-debug.js:35]: For consistency, shouldn't this be `INITIAL_BOUNDING_VOLUME_TYPE`?

File: examples/experimental/i3s-17-and-debug/components/debug-panel.js [hunk 2]
```diff
@@ -281,12 +150,15 @@ export default class DebugPanel extends PureComponent {
   }
 
   _renderBoundingVolumeOptions() {
-    const {boundingVolumeColorMode, boundingVolume} = this.state;
+    const {
+      debugOptions: {boundingVolumeColorMode, boundingVolume},
+      onOptionsChange
```
Comment (ibgreen) [inline examples/experimental/i3s-17-and-debug/components/debug-panel.js:155]: `onDebugOptionsChange`?

**Review by ibgreen — APPROVED** [review]

File: examples/experimental/i3s-17-and-debug/app-debug.js (hunk 1 above)
Comment (dryabinin-actionengine) [reply examples/experimental/i3s-17-and-debug/app-debug.js]: Done

File: examples/experimental/i3s-17-and-debug/components/debug-panel.js (hunk 2 above)
Comment (dryabinin-actionengine) [reply examples/experimental/i3s-17-and-debug/components/debug-panel.js]: Renamed

**Pull request merged**